src/utils/sentry.js

```diff
@@ -0,0 +1,19 @@
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
+import { publicRuntimeConfig } from '../../next.config';
+
+const initializeSentry = () => {
+  Sentry.init({
+    environment: publicRuntimeConfig.environment,
+    dsn:
+      'https://0aa9d6e3306c46fc90eade0e05b955f8@o424400.ingest.sentry.io/5692505',
```

Comment:
Can you also make this configurable through an ENV variable? So that we can set it to an empty string locally. Otherwise Sentry will get flooded with errors caused by local WIP :)

Reply:
Yep of course

Reply:
Done